repo/packages/C/cassandra/0/config.json

```diff
@@ -79,19 +79,19 @@
       "description": "CPU shares to allocate to each Cassandra Server Instance.",
       "type": "number",
       "minimum": 0.0,
-      "default": 2.0
+      "default": 0.1
```

Comment:
0.1 Could be too small? Maybe 0.5 or 0.2? Just want to make sure that cpu throttling doesn't slow down C* to the point of the user finding it unusable.